herchelpackage/durante.py

```diff
@@ -0,0 +1,73 @@
+class FitnessTracker:
+    def __init__(self):
+        self.activities = {}
+
+    def log_activity(self, activity_name, minutes):
+        if activity_name in self.activities:
+            self.activities[activity_name] += minutes
+        else:
+            self.activities[activity_name] = minutes
+        print(f"Logged {minutes} minutes for {activity_name}. "
+              f"\nTotal: {self.activities[activity_name]} minutes.")
+
+    def remove_activity(self, activity_name):
+        if activity_name in self.activities:
+            del self.activities[activity_name]
+            print(f"Activity '{activity_name}' removed from the tracker.")
+        else:
+            print(f"Activity '{activity_name}' does not exist in the tracker.")
+
+    def display_activities(self):
+        if not self.activities:
+            print("No activities logged.")
+        else:
+            print("Current Activities:")
+            for activity_name, minutes in self.activities.items():
+                print(f"- {activity_name}: {minutes} minutes")
+
+    def check_activity(self, activity_name):
+        if activity_name in self.activities:
+            print(f"{activity_name}: {self.activities[activity_name]} "
+                  f"minutes logged.")
+        else:
+            print(f"No logs for {activity_name}.")
+
+    def clear_activities(self):
+        self.activities.clear()
+        print("All activities have been cleared.")
+
+    def menu(self):
```

Comment:
change your method name. Please refer to the coding guidelines:
https://docs.google.com/presentation/d/1INQFb9d0Ajyq-FGkKR2uY27VT6iscjGoj0KgjXOVlCo/edit#slide=id.g31e92de329f_8_0